meta-lmp-base/recipes-bsp/trusted-firmware-a/tf-a-fio_2.7.bb

```diff
@@ -0,0 +1,33 @@
+require tf-a-fio-common.inc
+
+SUMMARY = "Trusted Firmware-A FIO"
+SECTION = "bootloaders"
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://license.rst;md5=1dd070c98a281d18d9eefd938729b031"
+
+PROVIDES += "virtual/trusted-firmware-a"
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/tf-a-fio:"
```

Comment:
Should this be?
```
FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
```

Reply:
No need for FILESEXTRAPATHS:prepend on a normal recipe, just required for appends.